dev/NavigationView/TestUI/NavigationViewPage.xaml

```diff
@@ -51,6 +57,19 @@
                     </muxcontrols:NavigationViewItem.Content>
                 </muxcontrols:NavigationViewItem>
                 <muxcontrols:NavigationViewItem x:Name="IntegerItem" AutomationProperties.Name="Integer" Icon="Accept"/>
+                <muxcontrols:NavigationViewItem Content="E Menu Item" Icon="Home" x:Name="E"/>
+                <muxcontrols:NavigationViewItem Content="F Menu Item" Icon="Home" x:Name="F"/>
+                <muxcontrols:NavigationViewItem Content="G Menu Item" Icon="Home" x:Name="G"/>
+                <muxcontrols:NavigationViewItem Content="H Menu Item" Icon="Home" x:Name="H"/>
+                <muxcontrols:NavigationViewItem Content="I Menu Item" Icon="Home" x:Name="I"/>
+                <muxcontrols:NavigationViewItem Content="J Menu Item" Icon="Home" x:Name="J"/>
+                <muxcontrols:NavigationViewItem Content="K Menu Item" Icon="Home" x:Name="K"/>
+                <muxcontrols:NavigationViewItem Content="L Menu Item" Icon="Home" x:Name="L"/>
+                <muxcontrols:NavigationViewItem Content="M Menu Item" Icon="Home" x:Name="M"/>
+                <muxcontrols:NavigationViewItem Content="N Menu Item" Icon="Home" x:Name="N"/>
+                <muxcontrols:NavigationViewItem Content="O Menu Item" Icon="Home" x:Name="O"/>
+                <muxcontrols:NavigationViewItem Content="P Menu Item" Icon="Home" x:Name="P"/>
+                <muxcontrols:NavigationViewItem Content="Q Menu Item" Icon="Home" x:Name="Q"/>
             </muxcontrols:NavigationView.MenuItems>
```

Comment:
Don't add new functionality to NavigationViewPage. "NavigationView Test", "NavigationView Regression Test", and "NavigationView TopNav Test" nearly have the same xaml and code.